### PR TITLE
Add timeouts for http server

### LINKED ISSUE #1017: Configure connection timeouts on HTTP interface
Possibly the cause of #1006 
Also will be required to run the node publicly for the web and mobile wallet

Should be configured with timeouts similar to https://github.com/skycoin/skycoin-explorer/blob/master/explorer.go#L803-L809

### PR DESCRIPTION
**Pull request by stgleb**
closes #1017

### CODE REVIEW

**Comment by therealssj** [comment]
`make format`

**Comment by gz-c** [comment]
Needs to be configured with timeouts

**Comment by stgleb** [comment]
@gz-c Thank you, will be done

File: cmd/skycoin/skycoin.go [hunk 1]
```diff
@@ -293,6 +297,11 @@ var devConfig = Config{
 	// Wallets
 	WalletDirectory: "",
 
+	// Timeout settings for http.Server
+	ReadTimeout:  5,
```
Comment (gz-c) [inline cmd/skycoin/skycoin.go]: should multiply by time.Second here, not later

Comment (stgleb) [reply cmd/skycoin/skycoin.go]: Ok, thanks

File: cmd/skycoin/skycoin.go [hunk 2]
```diff
@@ -298,9 +298,9 @@ var devConfig = Config{
 	WalletDirectory: "",
 
 	// Timeout settings for http.Server
-	ReadTimeout:  5,
-	WriteTimeout: 10,
-	IdleTimeout:  60,
+	ReadTimeout:  5 * time.Second,
```
Comment (gz-c) [inline cmd/skycoin/skycoin.go]: Use same defaults from explorer.go:

ReadTimeout: 10 seconds
WriteTimeout: 60 seconds
IdleTimeout: 120 seconds

Comment (stgleb) [reply cmd/skycoin/skycoin.go]: Ok, thanks

**Comment by gz-c** [comment]
#1096

**Pull request closed without merging**